### PR TITLE
Rstudio default view and issue fixes

### PR DESCRIPTION
**Pull request by HannesBuseyne**

### CODE REVIEW

File: assets/js/dependencies/async_loader.js [hunk 1]
```diff
@@ -56,26 +56,19 @@
...
             }
             else{
               installedVersion=installed.split('‘')[1];
               installedVersion=installedVersion.substring(0,installedVersion.length-2).replace('-','.');
               var upToDate=true;
               for(var i=0;i<versions.length;i++){
                 if($(versions[i]).text().trim()!="@VERSION@"&& _versionCompare($(versions[i]).text().trim(),installedVersion)){
-                  $('.versionCheck').text('');
-                  $('.versionCheck').append('<a href="javascript:;" onclick="window.installpackage(\''+packageName+'\','+packageSource+');" class="btn btn-primary">Update</a>');
+                  $('.versionCheck').append('<a href="NULL" id="js-install" class="btn btn-primary js-external">Update</a>');
```
Comment (ludov04) [inline assets/js/dependencies/async_loader.js]: just leave out the href attribute here if we are not using it

File: views/topic/show.ejs [hunk 2]
```diff
@@ -86,7 +86,7 @@
         <pre><code class="R"><%- topic.examples %></code></pre>
       </section>
       <% if(inViewerPane){ %>
-        <a href="javascript:;" onclick="window.runExamples();" class="btn btn-primary">Run examples</a>
+        <a href="www.google.com" onclick="window.runExamples()" id="js-examples" class="btn btn-primary">Run examples</a>
```
Comment (ludov04) [inline views/topic/show.ejs]: I don't get why you link to google here

Comment (HannesBuseyne) [inline views/topic/show.ejs]: should be fixed : https://github.com/datacamp/rdocumentation-app/commit/6d7ecc31a3141265e9c7bfe42e103e6e8e953327

**Comment by ludov04** [comment]
@HannesBuseyne, seems good, gonna merge this

**Pull request merged**